cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -73,12 +73,18 @@ typedef uint32_t gpio_t;
  * @brief   Macro for accessing GPIO pins
  * @{
  */
-#ifdef CPU_FAM_SAML11
-#define GPIO_PIN(x, y)      (((gpio_t)(&PORT_SEC->Group[x])) | y)
-#elif defined(PORT_IOBUS)   /* Use IOBUS access when available */
+#ifdef MODULE_PERIPH_GPIO_FAST_READ
+#if defined(CPU_FAM_SAML11)
```

Comment:
How about

```suggestion
#if defined(PORT_IOBUS_SEC)
```

if more MCUs with the feature start popping up